src/Diff/ItemPatcher.php

```diff
@@ -18,9 +19,14 @@
 class ItemPatcher implements EntityPatcherStrategy {
```

Comment:
Uh, **this** is weird in my opinion. Item and Property are guaranteed to have a Fingerprint. We do not win anything by using the smaller patchers here.